react-paw-mailmerge/src/components/FileUpload.tsx

```diff
@@ -62,6 +71,27 @@ const FileUpload: React.FC = () => {
                             onChange={() => {setDataHasHeader(!dataHasHeader)}}
                     />
                 </Form.Group>
+                <Form.Group>
+                  <Form.Label>Empty field parsing option: </Form.Label>
+                  <div className='mb-3'>
+                    <Form.Check
+                      inline
+                      type='radio'
+                      value='ignore'
+                      label='Parse as empty string'
+                      checked={nullFieldOption === 'ignore'}
+                      onChange={(event) => {setNullFieldOption(event.target.value)}}
+                    />
+                    <Form.Check
+                      inline
+                      type='radio'
+                      value='replace'
+                      label='Replace with N/A'
+                      checked={nullFieldOption === 'replace'}
```

Comment:
I'm not quite sure how it's best to do this, but there's a change in my null handling branch that creates an enum for the nullFieldOption so that we can avoid having magic strings.

Reply:
Should we let this go to master and then I can fix it in my branch?